content/examples/flatcar-container-linux/_index.md

```diff
@@ -0,0 +1,204 @@
+
+```yaml
+version: '0.1'
+name: flatcar-install
+global_timeout: 1800
+tasks:
+- name: "flatcar-install"
+  worker: "{{.machine1}}"
+  volumes:
+  - /dev:/dev
+  - /statedir:/statedir
+  actions:
+  - name: "dump-ignition"
+    image: flatcar-install
+    command:
+    - sh
+    - -c
+    - echo 'eyJpZ25pdGlvbiI6eyJjb25maWciOnt9LCJzZWN1cml0eSI6eyJ0bHMiOnt9fSwidGltZW91dHMiOnt9LCJ2ZXJzaW9uIjoiMi4yLjAifSwibmV0d29ya2QiOnt9LCJwYXNzd2QiOnt9LCJzdG9yYWdlIjp7fSwic3lzdGVtZCI6e319' | base64 -d > /statedir/ignition.json
+  - name: "flatcar-install"
+    image: flatcar-install
+    command:
+    - /usr/local/bin/flatcar-install
+    - -s
+    - -i
+    - /statedir/ignition.json
+```
+
+### Ignition configurations
+
+Recommended way of configuring Flatcar is via Ignition file during installation.
+
+Example template supports providing Ignition configuration in a form of Base64 encoded string injected directly into the template.
+
+See [Flatcar documentation about provisioning](https://docs.flatcar-linux.org/os/provisioning/) for more details.
+
+To create sample configuration, create `config.yaml` file with the following content:
+
+```yaml
+passwd:
+  users:
+  - name: core
+    ssh_authorized_keys:
+    - YOUR_SSH_KEY
+```
+
+Then, use the following command to validate and encode your configuration:
+
+```shell
+ct -in-file config.yaml  | base64 -w0; echo
+```
+
+Returned string can be then put in the template, by replacing default `eyJpZ25pdGlvbiI6eyJjb25maWciOnt9LCJzZWN1cml0eSI6eyJ0bHMiOnt9fSwidGltZW91dHMiOnt9LCJ2ZXJzaW9uIjoiMi4yLjAifSwibmV0d29ya2QiOnt9LCJwYXNzd2QiOnt9LCJzdG9yYWdlIjp7fSwic3lzdGVtZCI6e319` Ignition configuration.
+
+### Target disk
+
+By default the template installs Flatcar to smallest disk found on the worker using `-s` flag.
+
+### Extra installation parameters
+
+See [Flatcar documentation](https://docs.flatcar-linux.org/os/installing-to-disk/) to see all available parameters.
+
+### Creating
+
+After modifying the example template, you need to create it in Tinkerbell using the following command:
+
+```shell
+tink template create -n flatcar-install -p /tmp/flatcar-install.tmpl
+```
+
+Please keep the `ID` of created template, as it will be used in next steps.
+
+## Creating workflow
+
+After creating the template, you need to create the workflow object. This can be done using the following command:
+
+```shell
+tink workflow create -t TEMPLATE_ID -r '{"machine1": "02:42:db:98:4b:1e"}'
+```
+
+This command will return you workflow ID, which is needed to check the status of the workflow. Template ID will no longer be used.
+
+## Provisioning
+
+With workflow created, you can boot/reboot your worker(s), so it picks up the new workflow.
+
+You can track the progress of workflow execution using the following commands:
+
+```shell
+tink workflow state WORKFLOW_ID
+tink workflow events WORKFLOW_ID
+```
+
+Once the workflow is finished, you can reboot the worker and it should now boot Flatcar Container Linux from disk.
```

Comment:
On Packet this is not true, for example, so maybe something like

```suggestion
Once the workflow is finished you can reboot the worker and, after configuring the worker to boot from disk, it should now boot Flatcar Container Linux.
```

Reply:
Sounds good, thanks!